src/assets/stylesheets/containers/experience.sass

```
@@ -1,111 +1,131 @@
     display: flex
-    justify-content: center
     flex-direction: column
-    &:not(:last-child)
-      margin-right: $m-lg
+    flex: 1 1 33%
+    justify-content: center
 
   &__description
-    font-size: $font-size-lg * 1.1
+    border-bottom: 1px solid rgba($color-gray, 0.3)
     color: rgba($color-gray, 0.5)
-    padding:
-      bottom: $m-xl
+    font-size: $font-size-xl
     margin:
-      top: $m-md
       bottom: $m-xl
-    border:
-      bottom: $m-xs/8 solid rgba($color-gray, 0.3)
-
-.card
-  background-color: $color-white
-  &--visible-on-top
-    @extend .card
-    position: relative
-    z-index: $z-higher
-    box-shadow: -$m-lg $m-lg $m-xl rgba($color-violet, 0.2)
+      top: $m-md
+    padding-bottom: $m-xl
 
-  &:last-child:nth-child(n+2)
-    margin-top: $m-xl
-    position: relative
-    left: 55px
+.divider
+  background-color: $brand-primary
+  border-radius: $border-radius-sm
+  border: $m-xs/8 solid $brand-primary
+  height: $m-xs
+  width: $m-xxl * 1.5
 
-  &__subcaption
-    color: $color-gray
-    font-size: $font-size-sm
-    padding: $m-sm
-    margin: $m-sm
-    text-transform: uppercase
-    border-bottom: $m-xs/4 solid $color-gray
-    display: inline-block
+.heading
+  font-size: $font-size-xl //* 2
 
-  &__caption
-    font-weight: bold
-    font-size: $font-size-lg * 1.3
-    padding:
-      left: $m-sm
-      bottom: $m-sm
+  &.-primary-branded
+    text-align: center
+    &::before
+      @extend .divider
+      content: ' '
+      display: block
+      margin: 0 auto $m-xl
 
-.divider
-  border: $m-xs/8 solid $color-violet
-  border-radius: 7px 7px 7px 7px
-  background-color: $color-violet
-  width: $m-xxl
-  height: $m-md / 1.9
+  &.-secondary-branded
+    &::before
+      @extend .divider
+      content: ' '
+      display: block
+      margin:
+        bottom: $m-xl
+        top: $m-xl
 
 .link-with-icon
-  font-size: $font-size-base * 1.1
   color: rgba($color-gray, 0.5)
   display: inline-block
-  position: relative
-  text-decoration: none
+  font-size: $font-size-base * 1.1
+  left: $m-xxl + $m-sm
   padding:
-    top: $m-sm
     bottom: $m-sm
-  left: $m-xxl + $m-sm
-  &::before
+    top: $m-sm
+  position: relative
+  text-decoration: none
+
+  &__icon
+    align-items: center
+    background-color: $brand-primary
+    border-radius: 50%
+    bottom: 0
+    display: flex
+    justify-content: center
+    margin-right: $m-sm
+    padding: $m-sm
     position: absolute
     right: 100%
     top: 0
-    content: '►'
-    display: block
-    color: $color-white
-    background-color: $color-violet
-    font-size: $font-size-lg
-    bottom: 0
-    border-radius: 50%
-    margin-right: $m-sm
-    padding:
-      top: $m-sm
-      bottom: $m-sm
-      left: $m-md
-      right: $m-md
+
++media-breakpoint-up(lg)
+  .card
+    margin-top: 0
+
+    &.-visible-on-top
+      box-shadow: -$m-lg $m-lg $m-xl rgba($brand-primary, 0.2)
+      position: relative
+
+    &:last-child:nth-child(n+2)
```

This would do:
```suggestion
    &:nth-child(2n)
```
So the second child only